app/src/components/Shared/ExpandItem.jsx

```diff
@@ -4,11 +4,14 @@ import classnames from 'classnames';
 import ExpandItemStyles from 'styles/components/shared/expand-item.scss';
 import { VelocityTransitionGroup } from 'velocity-react';
```

Comment:
Velocity + velocity-react adds a a lot of weight to the bundle size (around 300kb unminified), and it seems we are only using it in one place. Could you please have a look and see if we could easily replace with a simple CSS transition (only if it's super simple to do)